docs/Tutorial_DiscordMERLIN.ipynb

```diff
@@ -0,0 +1,1714 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #51.            if np.isinf(M_T[idx]):
</pre><p>I think we should remove this <code>break</code> section and, instead, in line 57 do something like:</p><pre class="ql-syntax" spellcheck="false">if discord_dist != np.inf and approx_P[idx] &gt; discord_dist:
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TDAmeritrade/stumpy/pull/505/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='33'/>

Reply:
@seanlaw 
should I care about returning -1 for `idx` and `nn_idx` when the `discord_dist` becomes `np.inf`? (because this is how STUMPY.stump behaves and I want to be consistent) 

In that case, I need to add an `if p.isinf(discord_dist)` after the `for-loop`. 

```
for idx in cands_idx:
    if discord_dist != np.inf and approx_P[idx] > discord_dist:
        # some code

if np.isinf(discord_dist):
   discord_idx = -1
   discord_nn_idx = -1

    return discord_idx, discord_dist, discord_nn_idx
```

Or, we can take care of it at the end before throwing out the result. 
Do you have any suggestion?

Reply:
<p>I think the code that you've provided seems reasonable</p>